lms/djangoapps/user_authn/views/deprecated.py

```diff
@@ -0,0 +1,161 @@
+            third_party_auth_error = _(text_type(msg))  # pylint: disable=translation-of-non-string
+            break
+
+    context = {
+        'login_redirect_url': redirect_to,  # This gets added to the query string of the "Sign In" button in the header
+        # Bool injected into JS to submit form if we're inside a running third-
+        # party auth pipeline; distinct from the actual instance of the running
+        # pipeline, if any.
+        'pipeline_running': 'true' if pipeline.running(request) else 'false',
+        'pipeline_url': auth_pipeline_urls(pipeline.AUTH_ENTRY_LOGIN, redirect_url=redirect_to),
+        'platform_name': configuration_helpers.get_value(
+            'platform_name',
+            settings.PLATFORM_NAME
+        ),
+        'third_party_auth_error': third_party_auth_error
+    }
+
+    return render_to_response('login.html', context)
+
+
+@ensure_csrf_cookie
+def register_user(request, extra_context=None):
+    """
+    Deprecated. To be replaced by :class:`user_authn.views.login_form.login_and_registration_form`.
+    """
+    # Determine the URL to redirect to following login:
+    redirect_to = get_next_url_for_login_page(request)
+    if request.user.is_authenticated:
+        return redirect(redirect_to)
+
+    external_auth_response = external_auth_register(request)
+    if external_auth_response is not None:
+        return external_auth_response
+
+    context = {
+        'login_redirect_url': redirect_to,  # This gets added to the query string of the "Sign In" button in the header
+        'email': '',
+        'name': '',
+        'running_pipeline': None,
+        'pipeline_urls': auth_pipeline_urls(pipeline.AUTH_ENTRY_REGISTER, redirect_url=redirect_to),
+        'platform_name': configuration_helpers.get_value(
+            'platform_name',
+            settings.PLATFORM_NAME
+        ),
+        'selected_provider': '',
+        'username': '',
+    }
+
+    if extra_context is not None:
+        context.update(extra_context)
+
+    if context.get("extauth_domain", '').startswith(settings.SHIBBOLETH_DOMAIN_PREFIX):
+        return render_to_response('register-shib.html', context)
+
+    # If third-party auth is enabled, prepopulate the form with data from the
+    # selected provider.
+    if third_party_auth.is_enabled() and pipeline.running(request):
+        running_pipeline = pipeline.get(request)
+        current_provider = provider.Registry.get_from_pipeline(running_pipeline)
+        if current_provider is not None:
+            overrides = current_provider.get_register_form_data(running_pipeline.get('kwargs'))
+            overrides['running_pipeline'] = running_pipeline
+            overrides['selected_provider'] = current_provider.name
+            context.update(overrides)
+
+    return render_to_response('register.html', context)
+
+
+@csrf_exempt
+@transaction.non_atomic_requests
+def create_account(request, post_override=None):
+    """
+    JSON call to create new edX account.
```

Comment:
Could we add something like that of above?
```
Deprecated. To be replaced by :class:`???`.
```
Also relates to my question noted elsewhere about [create_account in user_api](https://github.com/edx/edx-platform/blob/73af914cfc8c82fa7bd0fc8e5fd0c1eb9b81494c/openedx/core/djangoapps/user_api/accounts/api.py#L273).  What is this other `create_account` and how does it relate?

Reply:
Updated the docstring to include `Deprecated. Use RegistrationView instead.`

There's also a pre-existing `warnings.warn` statement below to this effect.